### PR TITLE
Move example yamls into an examples/ directory, exercise TASKCFG_x/POD_INSTANCE envvars

### PR DESCRIPTION
**Pull request by nickbp**

### CODE REVIEW

**Comment by nickbp** [comment]
retest this please

File: frameworks/helloworld/src/main/dist/svc.yml [hunk 1]
```diff
@@ -9,11 +9,11 @@ pods:
     tasks:
       server:
         goal: RUNNING
-        cmd: "echo hello >> hello-container-path/output && sleep $SLEEP_DURATION"
+        cmd: "echo hello-$POD_INSTANCE_INDEX >> $OUTPUT_VOLUME/$OUTPUT_FILENAME && sleep $SLEEP_DURATION"
```
Comment (gabrielhartmann) [inline frameworks/helloworld/src/main/dist/svc.yml]: The changes here in svc.yml are illustrative, but I think a little complex for the default hello-world example.  Do you have objections to moving this to one of the example yamls, or a new example yaml (with an integration test that actually exercises code)?

Comment (nickbp) [inline frameworks/helloworld/src/main/dist/svc.yml]: I think it'd make sense to just put this in a separate yml that's exercised by an integration test. The `TASKCFG_*` stuff is only really useful in specific cases, and for newcomers it'd likely be confusing to have both env methods in the same place.

**Comment by nickbp** [comment]
Done

**Comment by nickbp** [comment]
retest this please

**Pull request merged**